code/modules/mob/delays.dm

```diff
@@ -5,7 +5,7 @@
 // Reduces duplicated code by quite a bit.
 /datum/delay_controller
 	// Delay clamps (for adminbus, effects)
-	var/min_delay = 1
+	var/min_delay = 0.001
```

Comment:
I was going to change it to .4 so that I could give back observers their old speed soon anyway, so I suggest that.